docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -5,6 +5,18 @@ description: A software-defined asset is a description of how to compute the con
 
 # Software-Defined Assets
 
+<Note>
```

Comment:
The way this ends up rendered comes out a little weird: the "Prefer videos" gets similar formatting to the links. Not a blocker, but maybe worth addressing if easy.

<img width="758" alt="image" src="https://user-images.githubusercontent.com/654855/189001113-4b956e27-a212-49d3-b319-08c04e591bc6.png">

Reply:
Agreed. I'll remove the bold for now and come back to it for the styling - there are a few general changes we could make to improve our text formatting, this being one of them.